README

```diff
@@ -279,8 +279,7 @@ Known issues
 Sieve
 -----
 
-Most open issues are outlined in the TODO file. The more generic ones are (re-)
-listed here:
+The more generic ones are (re-)listed here:
```

Comment:
I think we should just remove this Known issues entirely. The managesieve's doc.dovecot.org link is also weird why it's under Known issues. We should just have the `For up to date information please visit https://doc.dovecot.org/` in the beginning of the README.